src/screens/BillChecking/Masters/VendorMaster.jsx

```diff
@@ -50,9 +50,9 @@ function VendorMaster({ match }) {
   const [Panuppercase, SetPanUpeeerCase] = useState();
   const [ifscodeUppercase, setIfsccodeUppercase] = useState();
   const [succes, setSucces] = useState();
-  const [error, setError] = useState();
+  const [error, setError] = useState(null);
 
-  const [notify, setNotify] = useState();
+  const [notify, setNotify] = useState(null);
```

Comment:
Why toast and Alert Component both are used on same page?
Kindly avoid such thing in future.